src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1022,8 +1021,14 @@ def _matrices(self, file: api.File) -> MutableJSON:
                 'version': file.manifest_entry.version,
                 'name': file.manifest_entry.name,
                 'size': file.manifest_entry.size,
+                # FIXME: Remove deprecated field 'matrix_cell_count'
```

Comment:
Are you sure the snake-case version is deprecated? This is an index document so it *should* use snake case.

Reply:
The `matrices` and `contributor_matrices` inner entities are not currently translated like the other inner entities are, instead, they are passed along to `make_stratification_tree` and used without modification as leaf nodes (small exception, the `strata` key/value is removed from the final output).

I will adjust the processing of `matrices` and `contributor_matrices` in `hca_response_v5.py`  to match the translation behavior the other inner entities go through.